_data/lang/it/navigation.yml

```diff
@@ -0,0 +1,54 @@
+- title: Inizia da Qui
+  subfolderitems:
+  - page: cos'è Monero?
+    url: get-started/what-is-monero
+  - page: Utilizzo
+    url: get-started/using
+  - page: Adozione
+    url: get-started/accepting
+  - page: Contribuire
+    url: get-started/contributing
+  - page: Mining
+    url: get-started/mining
+  - page: FAQ
+    url: get-started/faq
+- title: Downloads
+  url: downloads/
+- title: Notizie Recenti
+  subfolderitems:
+    - page: Tutti i Post
+      url: blog
+    - page: Missive
+      url: blog/tags/monero%20missives.html
+    - page: Logs dei Meetings
+      url: blog/tags/dev%20diaries.html
+    - page: Uscite
+      url: blog/tags/releases.html
+- title: Community
+  subfolderitems:
+  - page: Team
+    url: community/team
+  - page: Hangouts
+    url: community/hangouts
+  - page: Eventi
+    url: community/events
+  - page: Sponsorships
+    url: community/sponsorships
+  - page: Mercanti
+    url: community/merchants
+- title: Risorse

```

Comment:
Questa sezione andrebbe uniformata relativamente alla stessa su [_data/lang/it/footer-1.yml](https://github.com/monero-project/monero-site/pull/593/files#diff-f501a8539a64ace8046295bf3f13a5cd).

Reply:
This section should be standardized relative to the same on  [_data/lang/it/footer-1.yml](https://github.com/monero-project/monero-site/pull/593/files#diff-f501a8539a64ace8046295bf3f13a5cd).